geonode/themes/models.py

```diff
@@ -97,7 +97,6 @@ class GeoNodeThemeCustomization(models.Model):
     contact_city = models.CharField(max_length=255, null=True, blank=True)
     contact_street = models.CharField(max_length=255, null=True, blank=True)
     contact_postal_code = models.CharField(max_length=255, null=True, blank=True)
-    contact_city = models.CharField(max_length=255, null=True, blank=True)

```

Comment:
Are we sure we are not using this variable?

Reply:
It is duplicated, it is defined twice. There are multiple fields in the models that are defined multiple times.